containers/images/candlepin/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:7
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release && \
+    yum -y install https://fedorapeople.org/groups/katello/releases/yum/nightly/katello/el7/x86_64/katello-repos-latest.rpm && \
+		yum -y install candlepin openssl qpid-tools
```

Comment:
Usually containers will have a `yum clean all` or similar command to reduce the layer size after this step.